src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -1387,6 +1387,9 @@ struct Gcx
                     auto pool = list.pool;
                     auto biti = cast(size_t)(cast(void*)list - pool.baseAddr) >> Pool.ShiftBy.Small;
                     assert(pool.freebits.test(biti));
+                    ppprev = pprev;
+                    pprev = prev;
+                    prev = list;
```

Comment:
This seems unrelated to this commit (use clone() instead of fork()).

Reply:
Thanks @llucax for review. Indeed this crept in during debugging, but I think it must have been inadvertently been removed at some time as the variables are still declared.